### PR TITLE
Update redirects-transition.conf

### PR DESCRIPTION
**Pull request by kathycarothers**
I have updated the presidential matching funds submission pages from transitions to wagtail. The need to be reviewed and deployed.

### CODE REVIEW

**Review by dorothyyeager — APPROVED** [review]
This looks great to me Kathy. Thank you so much for working on this particular project.

**Comment by bmathesonFEC** [comment]
I just made a commit to Kathy's branch off of dev. Added a few previously missed redirects and new redirects identified in the 7/13 transition inventory meeting.

File: redirects-transition.conf [hunk 1]
```diff
@@ -422,7 +429,12 @@ rewrite ^/law/litigation_CCA_X.shtml https://www.fec.gov/legal-resources/court-c
 
 # law/cfr/ redirects
 rewrite ^/law/cfr/cfr.shtml https://www.fec.gov/regulations redirect;
-rewrite ^/law/cfr/ej_citation.shtml https://www.fec.gov/legal-resources/regulations/explanations-and-justifications/explanations-and-justifications-citation-index/ redirect;
+rewrite ^/law/cfr/cfr_2008.pdf https://www.govinfo.gov/content/pkg/CFR-2008-title11-vol1/pdf/CFR-2008-title11-vol1.pdf redirect;
+rewrite ^/law/cfr/cfr_2009.pdf https://www.govinfo.gov/content/pkg/CFR-2009-title11-vol1/pdf/CFR-2009-title11-vol1.pdf redirect;
+rewrite ^/law/cfr/11_cfr.pdf https://www.govinfo.gov/content/pkg/CFR-2010-title11-vol1/pdf/CFR-2010-title11-vol1.pdf redirect;
+rewrite ^/law/cfr/2014cfr.pdf https://www.govinfo.gov/content/pkg/CFR-2014-title11-vol1/pdf/CFR-2014-title11-vol1.pdf redirect;
+rewrite ^/law/cfr/11cfr2015.pdf https://www.govinfo.gov/content/pkg/CFR-2015-title11-vol1/pdf/CFR-2015-title11-vol1.pdf redirect;
+xrewrite ^/law/cfr/ej_citation.shtml https://www.fec.gov/legal-resources/regulations/explanations-and-justifications/explanations-and-justifications-citation-index/ redirect;
```
Comment (patphongs) [inline redirects-transition.conf:437]: @kathycarothers Please change from `xrewrite` to `rewrite` at the beginning of this line.

**Review by patphongs — CHANGES_REQUESTED** [review]
@kathycarothers Just one minor change:

**Review by patphongs — APPROVED** [review]
Looks good, thanks @kathycarothers!

**Pull request merged**